stdlib/_interpqueues.pyi

```diff
@@ -0,0 +1,16 @@
+from typing import SupportsIndex
+
+class QueueError(RuntimeError): ...
+class QueueNotFoundError(QueueError): ...
+
+def bind(qid: SupportsIndex) -> None: ...
+def create(maxsize: SupportsIndex, fmt: SupportsIndex) -> int: ...
+def destroy(qid: SupportsIndex) -> None: ...
+def get(qid: SupportsIndex) -> tuple[object, int]: ...
```

Comment:
`object` should probably `Any` here, since called will often know the type, so `isinstance` checks could be bothersome.

Reply:
Yeah, I suppose I was just trying to be as narrow as possible. My guess is that using `object` is mostly going to make more sense for arguments the majority of the time, and there aren't a lot of reasons we'd want to use it in return types? Would be curious if there's any past discussion on this.

Reply:
Changed: https://github.com/python/typeshed/pull/12240/commits/a77bcf88a8fe1b4e510dc065068621080cd1d17e